### ISSUE TITLE
GOSI10: minor changes package

### ISSUE THREAD

**Issue by ukmo-cguiavarch** [JMMP-Group/GO_coordination#18]
Issue to reference all the smaller changes that we plan to implement in GOSI10. 

1. Internal wave mixing: implementing  the De Lavergne scheme instead of Simmons
2. Wind stress: implementing the current feedback option (Renault et al) instead of the relative wind stress
3. Chlorophyll: using a 2D map climatology instead of constant value  ( see https://code.metoffice.gov.uk/trac/gmed/ticket/631 )
4. ...

1 and 2 need to be implemented before we upgrade to NEMO5 () as it will allow to simplify the GOSI branch as Simmons et al and the relative wind stress are not available in the trunk at NEMO 4.2 and above.

**Comment by oceandie** [comment]
Should we include the inclusion of key_qco here? ... according to these [notes](https://github.com/JMMP-Group/GO_coordination/issues/2#issuecomment-2092911722):

_We should test the QCO vertical coordinate (key_qco) at v4.2.2 before porting to NEMO 5, as VVL will no longer be  available_

**Comment by ukmo-cguiavarch** [comment]
> Should we include the inclusion of key_qco here? ... according to these [notes](https://github.com/JMMP-Group/GO_coordination/issues/2#issuecomment-2092911722):
> 
> _We should test the QCO vertical coordinate (key_qco) at v4.2.2 before porting to NEMO 5, as VVL will no longer be available_

Yes, it was originally on the list but we decided it needed to be tested with general vertical coordinates first. If it is not available at NEMO5 , then we need add it at GOSI10p2 which will include the minor changes listed above. Isabella has already tested it for z coordinates as it was used by mistake in the first GOSI10p0.0. There is still the question of testing it with general vertical coordinates.

**Comment by ukmo-cguiavarch** [comment]
**Internal waves mixing [De Lavergne et al (2020)](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2020MS002065):**

Adam has interpolated the [De Lavergne et al (2020) input files](https://www.seanoe.org/data/00619/73082/) for eORCA025 (@4.2) . 

The file is available on monsoon:  /projects/jmmp/atblak/ancil/eORCA025/zdfiwm_forcing_eORCA025.nc

_**namelist settings are:**_
&namzdf_iwm
   ln_mevar    = .true.,   !  variable (T) or constant (F) mixing efficiency
   ln_tsdiff   = .true.,    !  account for differential T/S mixing (T) or not (F)

   cn_dir      = './INPUT/'      !  root directory for the iwm data location
   !           !  file name              ! frequency (hours) ! variable  ! time interp.!  clim  ! 'yearly'/ ! weights filename ! rotation ! land/sea mask !
   !           !                         !  (if <0  months)  !   name    !   (logical) !  (T/F) ! 'monthly' !                  ! pairing  !    filename   !
   sn_mpb      = 'zdfiwm_forcing_eORCA025.nc'  , -12.         , 'power_bot' , .false.  , .true. , 'yearly' , '' , ''  , ''
   sn_mpc      = 'zdfiwm_forcing_eORCA025.nc'  , -12.         , 'power_cri' , .false.  , .true. , 'yearly' , '' , ''  , ''
   sn_mpn      = 'zdfiwm_forcing_eORCA025.nc'  , -12.         , 'power_nsq' , .false.  , .true. , 'yearly' , '' , ''  , ''
   sn_mps      = 'zdfiwm_forcing_eORCA025.nc'  , -12.         , 'power_sho' , .false.  , .true. , 'yearly' , '' , ''  , ''
   sn_dsb      = 'zdfiwm_forcing_eORCA025.nc'  , -12.         , 'scale_bot' , .false.  , .true. , 'yearly' , '' , ''  , ''
   sn_dsc      = 'zdfiwm_forcing_eORCA025.nc'  , -12.         , 'scale_cri' , .false.  , .true. , 'yearly' , '' , ''  , ''
/

**Comment by atb299** [comment]
The zdfiwm input file is now available at all 3 resolutions here: /projects/jmmp/atblak/ancil/eORCAxx

I also copied across the original file that I interpolated and the weights for the eORCA025 and eORCA12 grids.

**Comment by ukmo-cguiavarch** [comment]
**GOSI10p2.0**(u-dh157) includes the following changes compared to GOSI10p1.2:
 

1. Internal wave mixing: implementing  the De Lavergne scheme instead of Simmons
2. Wind stress: implementing the current feedback option (Renault et al) instead of the relative wind stress
3. Chlorophyll: using a 2D map climatology instead of constant value  ( see https://code.metoffice.gov.uk/trac/gmed/ticket/631 )
4. Remove partial slip from SO package as it is not required with the new bathymetry.
5. Add key_qco 
 
The changes in GOSI10p2.0 compared to GOSI10p1.2 are:

- VALSO: reduction of ACC transport associated with increased Weddell gyre transport

- VALNA: significant change in SSS (fresher) and MLD(shallower) in Labrador sea

- VALTRANS: reduction of transport in Denmark Strait and Faroe Bank channel, increase in Lombok strait

- [marine_assess u-157 vs. udg878](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ocean/u-dh157_nemo_vs_u-dg878_nemo/assess.html): large changes in salinity in Arctic (fresher in the [surface layers](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ocean/u-dh157_nemo_vs_u-dg878_nemo/u-dh157_nemo_u-dg878_nemo_djf_19951201_20051130_19951201_20051130_mean_sss_abs_polar_vs_WOA13v2_salinity_teos10_n_polar_ocean.png) , saltier in [subsurface](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ocean/u-dh157_nemo_vs_u-dg878_nemo/u-dh157_nemo_u-dg878_nemo_djf_19951201_20051130_19951201_20051130_mean_s_abs_polar_100m_vs_WOA13v2_salinity_teos10_n_polar_ocean.png) up to 300m)

- [seaice_assess u-157 vs. udg878](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ice/u-dh157_si3_vs_u-dg878_si3/assess.html): no significant changes

 
To understand the changes 3 sensitivity experiments have been carried to test changes individually:
a. revert to relative wind stress ([marine_assess u-di436 vs. u-dh157](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ocean/u-di436_nemo_vs_u-dh157_nemo/assess.html))
b. reintroduce partial slip in Southern  Ocean ([marine_assess u-di437 vs. u-dh157](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ocean/u-di437_nemo_vs_u-dh157_nemo/assess.html))
c. reverting to Simmons et al instead of De Lavergne ([marine_assess u-di438 vs. u-dh157](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ocean/u-di438_nemo_vs_u-dh157_nemo/assess.html))
 
From these experiment we can conclude that :

- the **reduction in ACC transport is linked to the removal of the partial slip in the Southern Ocean** (see VALSO below). At this point I suggest to not reintroduce the partial slip in the SO as more changes will be introduced in future prototypes likely to have an impact on ACC. We can revisit at a later time.

 
- the **large changes in Salinity at high latitudes** in the Northern Hemisphere (Lab Sea + Arctic) are **linked to the change of scheme for internal wave mixing** from Simmons et al to De Lavergne. Impact of Simmons et al on salinity [at the surface](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ocean/u-di438_nemo_vs_u-dh157_nemo/u-di438_nemo_u-dh157_nemo_djf_19951201_20051130_19951201_20051130_mean_sss_abs_polar_vs_WOA13v2_salinity_teos10_n_polar_ocean.png), [at 100m](https://gws-access.jasmin.ac.uk/public/jmmp/valid_ocean/u-di438_nemo_vs_u-dh157_nemo/u-di438_nemo_u-dh157_nemo_djf_19951201_20051130_19951201_20051130_mean_s_abs_polar_100m_vs_WOA13v2_salinity_teos10_n_polar_ocean.png). 

- The changes in transport (see VALTRANS below) are also linked to the internal wave mixing scheme.
 
- current feedback (Renault et al) has a limited impact.

**VALSO:**
![VALSO_GOSI10p2 0_tests](https://github.com/user-attachments/assets/0bac0877-2e6f-4be5-875f-eca1fb830461)

**VALNA:**
![VALNA_GOSI10p2 0_tests](https://github.com/user-attachments/assets/17412ed0-3ed0-46cc-8bd2-f3c9f2b89397)

**VALTRANS:**
![VALTRANS_GOSI10p2 0_tests](https://github.com/user-attachments/assets/2b02f886-91ab-4b5e-9dbb-50453a760997)
